### DISCUSSION TITLE
How to install the Laravel Vite Plugin?

### DISCUSSION THREAD

**Discussion by Vectorial1024** [laravel/framework#46260] [Q&A] [answered]
I have noticed that https://github.com/innocenzi/laravel-vite is currently deprecated, and that we should use https://github.com/laravel/vite-plugin . However, I don't seem to understand how to migrate. For example, I cannot find `laravel/vite-plugin` on Packagist, and the official Laravel github repo `composer.json` does not mention the vite plugin.

How may I install the Laravel vite integration? Or is it not yet released?

**Comment by rojtjo** [comment, marked as answer]
It's actually an npm package, not a composer package.

See:
https://github.com/laravel/laravel/blob/10.x/package.json#L9
https://www.npmjs.com/package/laravel-vite-plugin
